src/node/crypto.ts

```diff
@@ -10,7 +10,7 @@ import {
 
 export const getRandomValues = crypto.getRandomValues;
 export const subtle = crypto.subtle;
-export const timingSafeEqual = (crypto as any).timingSafeEqual;
+export const timingSafeEqual = (subtle as any).timingSafeEqual;
```

Comment:
Is the `as any` still needed? I imagine it was previously needed because it wasn't under the correct object

Reply:
It might not be needed. Not 100% certain ;-)

Reply:
Quick test and the build blows up without it ... for now, will land it like this an d we can revisit